pkg/interface/chat/src/js/components/lib/message.js

```diff
@@ -181,7 +181,15 @@ export class Message extends Component {
             <div className="hide-child" style={paddingTop}>
               <p className="v-mid f9 gray2 dib mr3 c-default">
                 <span
-                  className={contact.nickname ? null : "mono"}
+                  className={"pointer " + (contact.nickname ? null : "mono")}
+                  ref="author"
+                  onClick={() => {
+                    writeText(props.msg.author);
+                    this.refs.author.innerText = "Copied";
```

Comment:
This means that if the name is `mono`, the "Copied" will also be `mono`. We want the `Copied` to always be Inter, I think.